CIP-dapp-bridge-rework/LICENSE

```diff
@@ -0,0 +1,395 @@
+Attribution 4.0 International
```

Comment:
As also acceptable by commonly used standard for the Creative Commons license etc., and according to CIP practice (https://github.com/cardano-foundation/CIPs/blob/master/CIP-0001/README.md#licensing), this file should be eliminated and referred to by a link at the end of the CIP document.  The way you currently have it in the CIP body is proper & sufficient. 😎